src/main/java/org/example/hw3_2/Robot.java

```diff
@@ -0,0 +1,32 @@
+package org.example.hw3_2;
+
+public class Robot implements Participant {
+   private String name;
+    private int maxRunDistance;
+    private int maxJumpHeight;
+
+    // Конструктор класу
+    public Robot(String name, int maxRunDistance, int maxJumpHeight) {
+        this.name = name;
+        this.maxRunDistance = maxRunDistance;
+        this.maxJumpHeight = maxJumpHeight;
+    }
+
+    @Override
+    public void run(int distance) {
+        if (distance <= maxRunDistance) {
+            System.out.println(name + " пробіг " + distance + " м.");
```

Comment:
It would be nice to update the method to make it more particularly for the robot in this case
E.g. sout("Robot" + name + ...)

Reply:
Done